ddp-workspace/projects/ddp-singular/src/app/components/pages/participant-list/participant-list.component.html

```diff
@@ -59,7 +59,19 @@ <h1 class="title">
                   <ng-container *ngTemplateOutlet="chevronRightSvg"></ng-container>
               </div>
           </button>
-          <p class="share-project" [innerHTML]="'ParticipantsList.ShareProject' | translate"></p>
+          <div class="family-enrollment-section" *ngIf="featureFlag_DDP_8506">
+            <img class="enrollment-image" src="/assets/images/step-1_DDP-8404.png" alt="enrollment image">
+            <div class="family-enrollment-description">
+              <h3 class="title-enrollment-message">Family Enrollment Message</h3>
```

Comment:
Can you please add the title translation?

Reply:
done